pages/checkout.tsx

```diff
@@ -0,0 +1,53 @@
+import {
+  FaAddressBook,
+  FaAtlas,
+  FaBus,
+  FaCommentsDollar,
+  FaHandshake,
+} from 'react-icons/fa';
+import { Box, Button, Flex, Stack } from '@chakra-ui/react';
+import { Step, Steps, useSteps } from 'chakra-ui-steps';
+
+import BuyerContact from '@packages/components/BuyerContact';
+import BuyerAddress from '@packages/components/BuyerAddress';
+import BuyerDelivery from '@packages/components/BuyerDelivery';
+import BuyerPayment from '@packages/components/BuyerPayment';
+import BuyerOrder from '@packages/components/BuyerOrder';
+import Footer from '@packages/components/Footer';
+
+const steps = [
+  { label: 'Contato', content: <BuyerContact />, icon: FaAddressBook },
+  { label: 'Endereço', content: <BuyerAddress />, icon: FaAtlas },
```

Comment:
Acredito que não precisamos do step `Endereço`. Podemos pedir o endereço no step `Envio` caso o usuário selecione a opção de envio pelo correio. O que vc acha?

Reply:
Mas de todo jeito se alguém da comunidade for entregar vai precisar do endereço não ?

Reply:
hum... talvez não. Acho que nessa opção as pessoas vão combinar via msg como será a entrega

Reply:
Blz vou começar a corrigir